iOS/DuckDuckGo/SettingsLegacyViewProvider.swift

```diff
@@ -81,6 +81,13 @@ class SettingsLegacyViewProvider: ObservableObject {
         return storyboard.instantiateViewController(withIdentifier: identifier)
     }
 
+    private func instantiateAppIconController(onChange: @escaping (AppIcon) -> Void) -> UIViewController {
```

Comment:
Could you rename it makeAppIconViewController?

Reply:
I'm keeping it consistent with other methods here, so would prefer to keep it like this.